src/main/java/orbis/loader/GhidraOrbisElfLoader.java

```diff
@@ -114,8 +116,9 @@ private static OrbisElfHeader getElfHeader(ByteProvider provider)
 			.filter(o -> o.getName().equals("Image Base"))
 			.findFirst()
 			.orElseThrow();
-		long base = NumericUtilities.parseLong((String) baseOption.getValue());
-		if (base == 0) {
+		BigInteger zero = new BigInteger("0");
+		BigInteger base = NumericUtilities.parseHexBigInteger((String) baseOption.getValue());
```

Comment:
This is how it gets validated by the `ElfLoaderOptionsFactory`. See if this works instead.
https://github.com/NationalSecurityAgency/ghidra/blob/fe7585b49207b440f99927a1b46d20a878ad17e7/Ghidra/Features/Base/src/main/java/ghidra/app/util/opinion/ElfLoaderOptionsFactory.java#L186

Reply:
If you don't want to change it from using `BigInteger` then you can at least use `BigInteger.ZERO` instead of constructing a new one.

Reply:
I'll make the changes tomorrow.